app/inputs/active_admin/inputs/select_input.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveAdmin::Inputs::SelectInput < Formtastic::Inputs::SelectInput
+  def input_html_options
+    super.merge(data: { tags: !!@options[:tags] })
```

Comment:
Is the `!!` only to allow for nil -> false as a default or is there another reason for it?

Reply:
Yes, that's the only reason. Thinking about it, I could use `present?` instead. What do you think?

Reply:
I think using `present?` would be slightly better as it's a bit more explicit

Reply:
Agree, I'll rebase and add the change